prow/github/client.go

```diff
@@ -2315,6 +2315,112 @@ func (c *client) GetRepos(org string, isUser bool) ([]Repo, error) {
 	return repos, nil
 }
 
+// GetRepoNames returns names of all repos in an org.
+//
+// This call uses multiple API tokens when results are paginated.
+//
+// See https://docs.github.com/en/graphql/reference/objects#repository
+func (c *client) GetRepoNames(org string, isUser bool) ([]Repo, error) {
```

Comment:
Maybe this would be the good opportunity to get rid of the `isUser` modal and simply provide `GetRepoNamesForOrg` and `GetRepoNamesForUser` to stress out the fact that these calls are different in GH API.

Reply:
Updated to do this, I will need to hunt down where all these are called and replace them. Before had wanted to make sure everyone was ok with the structure of the code. I am getting the feel for Golang still and was not sure if I should put some of these in `types.go`